app/src/main/java/com/simplecity/amp_library/glide/utils/BitmapAndSize.java

```diff
@@ -19,9 +19,7 @@ public boolean equals(Object o) {
 
```

Comment:
I'd prefer not to make this change. 'Simplifying' the conditional statement generally makes it harder to read for equals() method.